config/maintainers.json

```diff
@@ -8,8 +8,8 @@
       "name": "Erik Schierboom",
       "link_text": "My blog",
       "link_url": "http://www.erikschierboom.com/",
-      "avatar_url": "https://nl.gravatar.com/userimage/27030165/76c834a4087478d5f9ff6ff8c6c1429e.jpg",
-      "bio": "I am a developer with a passion for learning new languages. Scala was my first functional language, and I've been hooked ever since!"
+      "avatar_url": null,
+      "bio": null
```

Comment:
Was this deliberately edited as part of this PR? (cc @ErikSchierboom)

Reply:
I don't think it was. Let me re-add it.

Reply:
@ErikSchierboom, sorry for that. :man_facepalming: Thanks for noticing, @iHiD